src/components/input/input.html

```diff
@@ -18,6 +18,7 @@
              [spellcheck]="spellcheck"
              [attr.maxlength]="maxLength"
              [type]="type"
+             [name]="name"
```

Comment:
For the time being, this actually needs to be `attr.name` so that Angular does add an `name="null"` attribute when it's absent.